docs/components/board/_index.md

```diff
@@ -0,0 +1,1534 @@
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.gpio_pin_by_name).
+
+```python
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+# Get the GPIOPin with Pin Number 15 (GPIO 22 on Raspberry Pi 4).
+pin = await my_board.GPIO_pin_by_name(name="15")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `name` [(string)](https://pkg.go.dev/builtin#string): Pin number (NOT GPIO number) of the GPIOPin you want to retrieve.
+Refer to the pinout diagram and data sheet of your [board model](#configuration) for pin numbers and orientation.
+
+**Returns:**
+
+- `pin` [(GPIOPin)](https://pkg.go.dev/go.viam.com/rdk/components/board#GPIOPin): An interface representing an individual GPIO pin on the board.
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Board).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the GPIOPin with Pin Number 15 (GPIO 22 on Raspberry Pi 4).
+pin, err := myBoard.GPIOPinByName("15")
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### AnalogReaderNames
+
+Get the name of every [`AnalogReader`](#analogs) configured and residing on the board.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- `names` [(List[str])](https://docs.python.org/3/library/stdtypes.html#typesseq-list): An list containing the `"name"` of every analog reader [configured](#configuration) on the board.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.analog_reader_names).
+
+```python
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+# Get the name of every AnalogReader configured on the board.
+names = await my_board.analog_reader_by_name()
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- `names` [([]string)](https://go.dev/tour/moretypes/7): An slice containing the `"name"` of every analog reader [configured](#configuration) on the board.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Board).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the name of every AnalogReader configured on the board.
+names := myBoard.AnalogReaderNames()
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### DigitalInterruptNames
+
+Get the name of every [`DigitalInterrupt`](#digital_interrupts) configured on the board.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- `names` [(List[str])](https://docs.python.org/3/library/stdtypes.html#typesseq-list): An list containing the `"name"` of every interrupt [configured](#configuration) on the board.
```

Comment:
```suggestion
- `names` [(List[str])](https://docs.python.org/3/library/stdtypes.html#typesseq-list): An list containing the `"name"` of every interrupt [configured](#configuration) on the board.
```
```suggestion
- `names` [(List[str])](https://docs.python.org/3/library/stdtypes.html#typesseq-list): A list containing the `"name"` of every interrupt [configured](#configuration) on the board.
```

Reply:
I think this one might have been an accident with github being buggy, I can't see a difference in the code!